javaScript/popup.js

```diff
@@ -0,0 +1,301 @@
+    } else {
+      popupTech2 += `<li >${works[i].popuplanguage[j]}</li>`;
+    }
+  }
+
+  for (let k = 0; k < works[i].experiance.length; k += 1) {
+    experianceList
+      += k === 0
+        ? `<li><h3>${works[i].experiance[k]}</h3></li>`
+        : `<li class="counter"><img src="images/Counter.png" alt="Backend dev">${works[i].experiance[k]}</li>`;
+  }
+  body.style.overflow = 'hidden';
+  document.querySelector('.project-container').innerHTML = `
+    <div class='popup-item' id='works-${works[i].id}'>
+    <div class='popup-header'>
+    <div class="popup-header-left">
+    <h2>${works[i].name}</h2>
+    <ul class="container">
+    ${experianceList}
+    </ul>
+    </div> 
+    <i onclick="closePopupModal()" class="fa-solid fa-xmark close-project" id="close"></i>
+    </div>
+    <img src='${works[i].image}' class="hero-image">
+    <div class='description'> 
+    <div>
+    </div>
+    <div class="popup-footer">
+    <p>${works[i].popDescription}</p>
+    <div class="popup-footer-right">
+    <ul class='languages'>
+    ${popupTech1}
+   </ul>
+   <ul class='languages'>
+   ${popupTech2}
+  </ul>
+  <div class="btn-tags">
+  <button class="see-more"><a class="see"  href="${works[i].seeLive}">See Live</a><img src="images/icon.png" />
+  </button>
+  <button  class="see-more"><a class="see" href="${works[i].seeSource}">See Source</a><img src="images/vector-github.png" />
+  </button>
+  </div>
+    </div> 
+    </div>
+    </div>
+    </div>
+   `;
+
+  popModel.style.display = 'block';
+};
+
+const popMobModelMain = document.querySelector('.mobile-project-container');
+const displayMobPopup = (i) => {
+  let popupTech1 = '';
+  let experianceList = '';
+
+  for (let j = 0; j < works[i].popuplanguage.length; j += 1) {
+    if (j <= 3) {
+      popupTech1 += `<li >${works[i].popuplanguage[j]}</li>`;
+    }
+  }
+
+  for (let k = 0; k < works[i].experiance.length; k += 1) {
+    experianceList
+      += k === 0
+        ? `<li><h3>${works[i].experiance[k]}</h3></li>`
+        : `<li class="counter"><img src="images/Counter.png" alt="Backend dev">${works[i].experiance[k]}</li>`;
+  }
+
+  body.style.overflow = 'hidden';
+
+  document.querySelector('.mobile-project-container').innerHTML = `
+  <div class='popup-item-mob ' id='works-${works[i].id}'>
+  <div class='popup-mob-header'>
+  <div class="popup-header-left">
+  <h2>${works[i].name}</h2>
+  <ul class="container">
+  ${experianceList}
+  </ul>
+  </div> 
+  <i onclick="closePopupMobModel()" class="fa-solid fa-xmark close-mob-project" id="close-mob"></i>
+  </div>
+  <img src='${works[i].image}' class="hero-image">
+  <div class='description'> 
+  <div>
+  </div>
+  <div class="popup-footer">
+  <p>${works[i].popDescription}</p>
+  <div class="popup-footer-right">
+  <ul class='languages'>
+  ${popupTech1}
+ </ul>
+
+<div class="btn-tags">
+<button class="see-more"><a class="see"  href="${works[i].seeLive}">See Live</a><img src="images/icon.png" />
```

Comment:
- [ ] OPTIONAL. Great job implementing the requested change here by the previous reviewer, but this has caused one Icon image link to break and not show, Kindly fix this.

Your design
![image](https://user-images.githubusercontent.com/81932478/169500054-0293b198-5ee9-4c4b-8f42-7ffe75fce372.png)